### PR TITLE
Use new signature of binstar_client.add_release if old one fails

### PR DESCRIPTION
**Pull request by jjhelmus**
Closes #84

### CODE REVIEW

**Comment by jjhelmus** [comment]
This is SciTools/conda-build-all#87 submitted against the conda-forge fork of `conda-build-all`

**Comment by jakirkham** [comment]
Toggling for CI.

**Comment by mwcraig** [comment]
I'll fix this up in a bit...

File: conda_build_all/build.py [hunk 1]
```diff
@@ -63,8 +63,15 @@ def upload(cli, meta, owner, channels=['main'], config=None):
...
-                        description='')
+
+        # The signature for add_release changed in anaconda-client 1.6.3.
+        # First try the old signature, and if that fails, use the new.
+        try:
+            cli.add_release(owner, package_name, version, requirements=[],
+                            announce=None, description='')
+        except TypeError:
+            cli.add_release(owner, package_name, version, requirements=[],
+                            announce=None, release_attrs={'description': ''})
```
Comment (jakirkham) [inline conda_build_all/build.py:74]: Maybe one day this should handle this metadata somehow? Not that it needs to today.

**Comment by jakirkham** [comment]
Thanks @mwcraig and @jjhelmus.

**Pull request merged**